Add(doc): custom operator image (#875)
DataDog/datadog-operator: docs/installation.md

See the [`kubectl` plugin doc](/docs/kubectl-plugin.md)

## Use a custom Datadog Operator image

> you can find the instruction to build a Datadog Operator custom container image based on an official release [custom-operator-image.md][9]

Reviewer: ```suggestion
See instructions to build a Datadog Operator custom container image on an official release in [Custom Operator container images][9].
```

> ### with the helm chart

Reviewer: ```suggestion
### Datadog Operator images with Helm charts

To use a Datadog operator image with a Helm chart:

```

See the [`kubectl` plugin doc](/docs/kubectl-plugin.md)

## Use a custom Datadog Operator image

> See instructions to build a Datadog Operator custom container image on an official release in [Custom Operator container images][9].

Reviewer 2: ```suggestion
See instructions to build a Datadog Operator custom container image based on an official release in [Custom Operator container images][9].
```

### Datadog Operator images with Helm charts

> To use a Datadog operator image with a Helm chart:

Reviewer 2: ```suggestion
To install a custom Datadog Operator image using the Helm chart, run the following:
```
